maintainers/maintainer-list.nix

```diff
@@ -1048,6 +1048,12 @@
     githubId = 453170;
     name = "Alastair Pharo";
   };
+  astralbijection = {
```

Comment:
Welcome!

It's custom to have the addition to the `maintainer-list.nix` in a separate commit (https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers).

Apologies for only commenting on such a trivial thing and not on the 'meat' of this PR, I'm afraid I don't feel qualified to judge the other changes...